demos/assets/ctxMenuFns.js

```diff
@@ -0,0 +1,25 @@
+window.dashCytoscapeFunctions = {};
+
+window.dashCytoscapeFunctions.add_2_nodes = function (event) {
```

Comment:
In as far as this should function as docs / recommended usage, it's better to do this similarly to how we document [clientside callbacks](https://dash.plotly.com/clientside-callbacks), which would be like:

```js
window.dashCytoscapeFunctions = Object.assign({}, window.dashCytoscapeFunctions, {
    add_2_nodes: function (event) { ... }
});
```

The reason being this form allows many such files to all add functions to the same namespace, as long as all the functions have different names, and it doesn't matter what order the files are included.